docs/book/supported_features_matrix.md

```diff
@@ -2,30 +2,27 @@
+| Dynamic Block PV support (`Read-Write-Once` Access Mode) | Yes                                                                             | Yes                                                           | Yes                                             | Yes                                                        | Yes                                                          | Yes                                                                 | Yes                                                                   |
+| Dynamic File PV support (`Read-Write-Many` Access Mode)  | No                                                                              | Yes (vSAN only)                                               | Yes (vSAN only)                                 | No                                                         | No                                                           | No                                                                  | No                                                                    |
+| Encryption support via VMcrypt                           | No                                                                              | Yes (Block Volume)                                            | Yes (Block Volume)                              | No                                                         | No                                                           | No                                                                  | No                                                                    |
+| Dynamic Virtual Volume (vVOL) PV support                 | No                                                                              | Yes                                                           | Yes                                             | No                                                         | No                                                           | No                                                                  | No                                                                    |
+| Offline Volume Expansion support (beta)                  | No                                                                              | Yes (Block Volume)                                            | Yes (Block Volume)                              | No                                                         | Yes (Block Volume)                                           | No                                                                  | Yes (Block Volume)                                                    |
```

Comment:
Offline volume expansion is not supported in Supervisor cluster in 7.0U1.

Reply:
fixed.